app/views/batches/_cherrypick_single_worksheet.html.erb

```diff
@@ -24,65 +30,70 @@
       <% if plate.buffer_required? %>
         <h3> Buffer Required </h3>
       <% end %>
-  <table class="plate_layout">
-    <thead>
-      <tr>
-      <% num_rows = plate.height -%>
-      <% num_columns = plate.width -%>
-      <th></th>
-      <% num_columns.times do |column| -%>
-        <th><%= column+1 %></th>
-      <% end -%>
-      <th></th>
-      </tr>
-    </thead>
-    <tbody>
-    <% plate_wells = plate.wells.index_by {|w| w.map.row_order } %>
-    <% num_rows.times do |row| -%>
-      <tr>
-        <% rowchar = (65+row).chr %>
-        <td><strong><%= rowchar %></strong></td>
-        <% (num_columns).times do |column| -%>
-          <% well = plate_wells[row*num_columns+column] -%>
-            <% if  well.present? -%>
-              <% request = indexed_requests[well.id]  %>
-              <% source_well = request&.asset %>
-              <% if source_well.present? && -%>
-                <% if source_well.plate.is_a?(ControlPlate) %>
-                  <td class="control">
-                <% elsif source_well.is_a?(Well) %>
-                  <td class="<%= source_plate_colour[source_well.plate.machine_barcode]%> ">
+    <table class="plate_layout">
+      <thead>
+        <tr>
+        <% num_rows = plate.height -%>
+        <% num_columns = plate.width -%>
+        <th></th>
+        <% num_columns.times do |column| -%>
+          <th><%= column+1 %></th>
+        <% end -%>
+        <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <% plate_wells = plate.wells.index_by {|w| w.map.row_order } %>
+        <% num_rows.times do |row| -%>
+          <tr>
+            <% rowchar = (65+row).chr %>
+            <td><strong><%= rowchar %></strong></td>
+            <% (num_columns).times do |column| -%>
+              <% well = plate_wells[row*num_columns+column] -%>
+                <% if  well.present? -%>
+                  <% request = indexed_requests[well.id]  %>
+                  <% source_well = request&.asset %>
+                  <% if source_well.present? -%>
+                    <% source_barcode = source_well.plate.machine_barcode %>
+                    <% well_is_in_this_pick = ( source_layout.has_key?(source_barcode) || (control_layout && control_layout.has_key?(source_barcode)) ) -%>
```

Comment:
Tempted to suggest we still show the other wells, just with a white background and gray text. Means you get an overview of what to expect on the whole plate.

Reply:
I went with this because 
a) if you do a pick with a partial plate, it doesn't seem to show the existing wells, just the ones you're picking, and
b) on the print preview, it doesn't show the colours on the wells

Do you know if the print preview not showing the colours is actually a bug?